src/app/cdk/panel/panel-data/panel-data.component.ts

```diff
@@ -16,6 +16,10 @@ export class PanelDataComponent implements OnInit, OnDestroy {
 
   @HostBinding('class.orc-font-body-small') fontSmall = true
   @HostBinding('class.border-bottom') borderBottomClass = false
+  @HostBinding('class.featured') featuredClass = false
+  @Input() set featured(featured: boolean) {
```

Comment:
(Suggestion optional): Ideally this "panel data input" should have a more generic name so it can be reused for other features without causing confusion.

For example, we have the input "borderBottom", which is an explicit name that clearly indicates its effect on a generic component. It's not tied to a single feature.